Cargo.toml

```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 
 [features]
 default = []
-bench = ["clap", "num_cpus", "log", "rand", "rayon", "zstd", "env_logger"]
+bench = ["clap", "num_cpus", "log", "rand", "historian", "time", "rayon", "zstd", "env_logger"]
```

Comment:
since I added historian as a dep on the main crate, this doesn't need to be in the feature array